pkg/providers/pricing/pricing.go

```diff
@@ -405,6 +405,7 @@ func populateInitialSpotPricing(pricing map[string]float64) map[string]zonal {
 }
 
 func (p *Provider) Reset() {
+	initialOnDemandPrices := lo.Assign(initialOnDemandPricesUS, initialOnDemandPricesUS)
```

Comment:
You're assigning US here twice so CN isn't included

Reply:
Since this wasn't caught by our testing, can we make sure to add a test to validate that we get the right data from the china region?

Reply:
Wow that's a miss...I guess that's why we do reviews. Are you thinking that test would just validate that the regions we expect data to exist for, `us-east-1`, `us-gov-east-1`, `us-gov-west-1`, and `cn-north-1`, all do?

Reply:
"If I do a get against pricing for each region without an update", do I get the data I expect from the price update map?